features/energy.py

```diff
@@ -19,15 +19,17 @@ def EnergyJob(crp, energy_options=None, extra_config=None, extra_post_config=Non
 
     port_name_mapping = {"energy": "energy"}
 
+    if "rtf" not in kwargs:
+        kwargs["rtf"] = 0.1
+    if "mem" not in kwargs:
+        kwargs["mem"] = 2
+
```

Comment:
no need to explicitly set them here (and in the other classes) as these values will already be the default in `FeatureExtractionJob`